examples/node/domstubs.js

```diff
@@ -135,6 +135,8 @@ function DOMElementSerializer(node) {
   this._childSerializer = null;
 }
 DOMElementSerializer.prototype = {
+  /* eslint-disable no-fallthrough */
```

Comment:
Please just add `/* falls through */` were applicable instead, rather than blanket disable like this.